tests/foreman/cli/test_errata.py

```diff
@@ -699,39 +729,40 @@ def test_positive_list_affected_chosts_by_erratum_restrict_flag(
 
     :CaseAutomation: Automated
     """
-
     # Uninstall package so that only the first errata applies.
     for host in errata_hosts:
+        host.execute(r'subscription-manager repos --enable \*')
+        host.execute(r'yum-config-manager --enable \*')
         host.execute(f'yum erase -y {REPO_WITH_ERRATA["errata"][1]["package_name"]}')
-
+        host.execute('subscription-manager repos')
+        sleep(20)
+        target_sat.cli.Host.errata_recalculate({'host-id': host.nailgun_host.id})
     # Create list of uninstallable errata.
     errata = REPO_WITH_ERRATA['errata'][0]
     uninstallable = REPO_WITH_ERRATA['errata_ids'].copy()
     uninstallable.remove(errata['id'])
-
     # Check search for only installable errata
     param = {
         'errata-restrict-installable': 1,
-        'content-view-id': module_cv.id,
         'lifecycle-environment-id': module_lce.id,
         'organization-id': module_entitlement_manifest_org.id,
         'per-page': PER_PAGE_LARGE,
     }
     errata_ids = get_errata_ids(param)
+    assert errata_ids, 'No installable errata found'
     assert errata['id'] in errata_ids, 'Errata not found in list of installable errata'
     assert not set(uninstallable) & set(errata_ids), 'Unexpected errata found'
 
     # Check search of errata is not affected by installable=0 restrict flag
     param = {
         'errata-restrict-installable': 0,
-        'content-view-id': module_cv.id,
         'lifecycle-environment-id': module_lce.id,
         'organization-id': module_entitlement_manifest_org.id,
         'per-page': PER_PAGE_LARGE,
     }
     errata_ids = get_errata_ids(param)
     assert set(REPO_WITH_ERRATA['errata_ids']).issubset(
-        errata_ids
+        set(errata_ids)
```

Comment:
Not necessary to cast, `issubset` takes any iterable. But also no harm to have it explicitly. Just saying.

Reply:
I did see an odd error here related to subset of the list where there were duplicate list entries. I wasn't too sure about it but can test more without `set(errata_ids)`

Reply:
Looks like it's passing without casting, i think the assertion fail here was related to the `module_ak_cv_lce` fixture that I'm no longer using.

Reply:
Might be related to Python version, in 3.10.11 I can see
```
set([2,3]).issubset([1,2,3,4])
Out[3]: True
set([2,3]).issubset([1,3,4])
Out[4]: False
```
Okay so keep it.

Reply:
Okay I'll put this back in to be safe